prototype/mixins.py

```diff
@@ -4,24 +4,28 @@
 from directory_components.helpers import SocialLinkBuilder
 
 from core.helpers import handle_cms_response
-from prototype.helpers import unprefix_prototype_url
+from prototype.helpers import unprefix_prototype_url, unslugify
 
 
-class GetCMSPageByFullPathMixin():
-
-    @cached_property
-    def page(self):
-        response = cms_api_client.lookup_by_full_path(
-            full_path=unprefix_prototype_url(self.request.path),
-            draft_token=self.request.GET.get('draft_token'),
-        )
-        return handle_cms_response(response)
+class Breadcrumbs:
 
     def get_context_data(self, *args, **kwargs):
+        parts = unprefix_prototype_url(self.request.path).split('/')
+        url_fragments = [part for part in parts if len(part) > 0]
```

Comment:
`[part for part in parts if part]` :boom:

Reply:
yeah i tried that and for some reason the first item that came out was always an empty string 🤷‍♀️

Reply:
obviously, as soon as I check it again it's working perfectly LOL. I will update this